src/commands/Google/gimage.ts

```diff
@@ -27,11 +27,11 @@ export default class extends RichDisplayCommand {
 
 	public async run(message: KlasaMessage, [query]: [string]) {
 		const [response, { items }] = await Promise.all([
-			message.sendEmbed(new MessageEmbed().setDescription(message.language.tget('SYSTEM_LOADING')).setColor(BrandingColors.Secondary)),
+			message.sendEmbed(new MessageEmbed().setDescription(message.language.get('SYSTEM_LOADING')).setColor(BrandingColors.Secondary)),
```

Comment:
we can just use send here no?
```suggestion
			message.send(new MessageEmbed().setDescription(message.language.get('SYSTEM_LOADING')).setColor(BrandingColors.Secondary)),
```

Reply:
We use `sendEmbed` for sending embeds, it's more explicit about what it sends.

Reply:
Sorry kyra I'm a bit confused

What does sendEmbed send

Reply:
Embeds.